src/app/store/request/request.slice.ts

```diff
@@ -0,0 +1,19 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+interface RequestState {
+  requestNetworkId?: string;
+}
+
+const initialState: RequestState = {
+  requestNetworkId: undefined,
+};
+
+export const requestSlice = createSlice({
+  name: 'request',
+  initialState,
+  reducers: {
+    requestNetworkActivated(state, action: PayloadAction<string>) {
+      state.requestNetworkId = action.payload;
+    },
+  },
+});
```

Comment:
Wondering if there's a simpler way to do this than creating a new slice just for this override behaviour, but request is too much of a generic name imo.  `networkOverride` would make more sense for now.